apps/frontend/src/components/Avatar.tsx

```diff
@@ -0,0 +1,40 @@
+import Image from 'next/image';
+import { useRouter } from 'next/navigation';
+
+import placeholderProfilePic from '../../public/profile_avatar_placeholder_large.png';
+
+export default function Avatar() {
+  const router = useRouter();
+  // TODO - Add logic to show the user's avatar here if they are logged in
+
+  // Maybe we could use shadcn's Avatar component here
+
+  const loggedIn = false;
```

Comment:
Alright. I have a solutiion for this using React's context api, but since we also need the info inside the server components, I think we'll have to use cookies for this. But yeah, this is a great topic to cover in a discord session.